lntest/itest/lnd_multi-hop_local_force_close_on_chain_htlc_timeout_test.go

```diff
@@ -129,7 +129,7 @@ func testMultiHopLocalForceCloseOnChainHtlcTimeout(net *lntest.NetworkHarness,
 		}
 	}
 
-	if _, err := net.SlowGenerate(defaultCSV); err != nil {
+	if _, err := net.SlowGenerate(defaultCSV - 1); err != nil {
```

Comment:
Same thing here that we already mined a block earlier confirming it? I don't see blocks being mined earlier though.